appengine/mailgun/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
I know this is part of the generator, but again, none of the other samples we'll be releasing use this. How strongly are you attached to this?